bin/convert2vulyk.py

```diff
@@ -0,0 +1,155 @@
+#!env python
+
+import argparse
+import json
+import logging
+import re
+import sys
+import time
+from collections import namedtuple
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.ERROR)
+
+BsfInfo = namedtuple('BsfInfo', 'id, tag, start_idx, end_idx, token')
+
+
+def parse_bsf(bsf_data: str) -> list:
+    """
+    Convert multiline textual bsf representation to a list of named entities.
+
+    :param bsf_data: data in the format 'T9	PERS 778 783    токен'. Can be multiple lines.
+    :return: list of named tuples for each line of the data representing a single named entity token
+    """
+    if len(bsf_data.strip()) == 0:
+        return []
+
+    ln_ptrn = re.compile(r'(T\d+)\s(\w+)\s(\d+)\s(\d+)\s(.+?)(?=T\d+\s\w+\s\d+\s\d+|$)', flags=re.DOTALL)
+    result = []
+    for m in ln_ptrn.finditer(bsf_data.strip()):
+        bsf = BsfInfo(m.group(1), m.group(2), int(m.group(3)), int(m.group(4)), m.group(5).strip())
+        result.append(bsf)
+    return result
+
+
+def convert_bsf_2_vulyk(text: str, bsf_markup: str) -> dict:
+    """
+    Given tokenized text and named entities in Brat standoff format, generate object
+    in the format compatible with Vulyk markup tool.
+    :param text: tokenized text (space as separator)
+    :param bsf_markup: named entities in Brat standoff format
+    :return: dict that can be directly converted to Vulyk json file
+    """
+    bsf = parse_bsf(bsf_markup)
+    ents = [[e.id, e.tag, [[e.start_idx, e.end_idx]]] for e in bsf]
+
+    idx = 0
+    t_idx = 0
+    s_offsets = []
+    t_offsets = []
+    if len(text) > 0:
```

Comment:
The God of tests said 'nope' :). 
Removing it fails the empty test by adding zero element instead of leaving it empty

```
   'protocol': 1,
   'relations': [],
-  'sentence_offsets': [],
+  'sentence_offsets': [[0, 0]],
?                       ++++++

   'source_files': ['ann', 'txt'],
   'text': '',
-  'token_offsets': [],
+  'token_offsets': [[0, 0]],
?                    ++++++

   'triggers': []}

```